tools/purge_ems.rb

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env ruby
+require File.expand_path('../config/environment', __dir__)
+require 'optimist'
+
+ARGV.shift if ARGV[0] == '--' # if invoked with rails runner
+batch_message = "For each relationship, the number of rows to be removed in a batch. A lower number may decrease memory usage but also take longer."
+opts = Optimist.options do
+  banner "Purge management system records in the background via the queue.\n\nUsage: ruby #{$PROGRAM_NAME} [options]\n\nOptions:\n\t"
+  opt :id,      "Management System id",                                         :type => :integer,                    :required => true
```

Comment:
Think rubocop has a good point about this one

Reply:
ok, I wanted to keep the different options split for readability but don't care too much.  Pushed.

Reply:
Oh was the `:type` aligned with something?  It looked like it probably was before but something else moved or changed.